### DISCUSSION TITLE
aliyundrive-webdav 后用Rclone挂载到本地不能写入吗？

### DISCUSSION THREAD

**Discussion by kitkw** [messense/aliyundrive-webdav#221] [Q&A]
直接拖拉文件进去会提示没有权限....
![6 085350](https://user-images.githubusercontent.com/31963066/147396283-1d8ad3d7-2aa4-47da-911e-16cd2e655f55.jpg)

**Comment by messense** [comment]
请提供详细复现步骤。

**Comment by kitkw** [comment]
![2](https://user-images.githubusercontent.com/31963066/147417588-2b0d9647-148d-444b-8469-35718fa06260.jpg)
![Screenshot 2021-12-18 160704](https://user-images.githubusercontent.com/31963066/147417602-303ce018-890a-408d-953a-c67dc3788c2e.jpg)
权限已经打开，其他目录能正常拖拉文件进出。只有aliyunwebdav目录拖拉文件进去就报权限
